### PR TITLE
[make:migration] Format the generated migration sql by passing `--formatted` to the command

### PR DESCRIPTION
**Pull request by StevenRenaux**
Hi,

Related to this #1336 I tried to work on this PR.

Now you can run `make:migration --formatted` to get a formatted SQL migration file.

### CODE REVIEW

File: src/Maker/MakeMigration.php [hunk 1]
```diff
@@ -88,6 +91,10 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         }
         // end 2.x support
 
+        if ($input->hasOption('formatted') && null !== $input->getOption('formatted')) {
```
Comment (mdoutreluingne) [inline src/Maker/MakeMigration.php:94]: There's a problem with this condition, for me it always `true` without having added the `--formatted` option. Here you just need to know if the option has been added, or not. So like this :

```suggestion
        if ($input->getOption('formatted')) {
```

Comment (StevenRenaux) [reply src/Maker/MakeMigration.php]: you right ;)

**Review by mdoutreluingne — APPROVED** [review]

File: src/Maker/MakeMigration.php [hunk 2]
```diff
@@ -70,6 +70,9 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
                 ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection name')
             ;
         }
+
```
Comment (ktherage) [inline src/Maker/MakeMigration.php:73]: the above help must certainly be updated. line 62.

**Comment by skigun** [comment]
This would be a nice to have 👍

File: tests/Maker/MakeMigrationTest.php [hunk 3]
```diff
@@ -131,5 +131,16 @@ public function getTestDetails(): \Generator
                 $this->assertStringNotContainsString('Success', $output);
             }),
         ];
+
+        yield 'it_generate_a_formatted_migration' => [$this->createMakeMigrationTest()
```
Comment (MaximePinot) [inline tests/Maker/MakeMigrationTest.php:135]:
```suggestion
        yield 'it_generates_a_formatted_migration' => [$this->createMakeMigrationTest()
```

**Review by jrushlow — APPROVED** [review]
thank you @StevenRenaux!

**Pull request merged**